charts/sn-platform/templates/broker/_broker.tpl

```diff
@@ -380,3 +380,20 @@ Define gcs offload options mounts
 {{ .Values.broker.serviceAccount.name }}
 {{- end -}}
 {{- end -}}
+
+{{/*
+pulsar service domain
+*/}}
+{{- define "pulsar.broker_service_domain" -}}
+{{- if .Values.ingress.broker.enabled -}}
+  {{- if .Values.ingress.broker.external_domain }}
```

Comment:
@fantapsody There is no `ingress.broker.external_domain` config on `values.yaml`